src/sentry/runner/commands/backup.py

```diff
@@ -18,7 +18,10 @@ def import_(src):
 
     for obj in serializers.deserialize("json", src, stream=True, use_natural_keys=True):
         if obj.object._meta.app_label not in EXCLUDED_APPS:
-            obj.save()
+            try:
+                obj.save()
+            except IntegrityError as e:
```

Comment:
We'll leave integrity errors to the user to handle by giving a warning so they're aware of the issue since it needs to be dealt with manually. We shouldn't error out the restore process and try to restore as much as we can.

Reply:
Is pkid collision the only possible IntegrityError? If there are others, do we want the same behavior for them?

Reply:
Okay let's put this in the code as a comment for our future selves to understand why it does this. :)

Reply:
good idea, done!